catalog/transformer/catalog-transformer-streaming-lib/src/test/java/org/codice/ddf/transformer/xml/streaming/lib/TestMetacardTypeRegister.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.transformer.xml.streaming.lib;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+import java.util.Set;
+
+import org.codice.ddf.transformer.xml.streaming.SaxEventHandlerFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.osgi.framework.BundleContext;
+
+import com.google.common.collect.ImmutableList;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.BasicTypes;
+
+public class TestMetacardTypeRegister {
+
+    MetacardTypeRegister metacardTypeRegister;
+
+    SaxEventHandlerFactory mockFactory;
+
+    @Before
+    public void setup() {
+        //mock factory
+        mockFactory = mock(SaxEventHandlerFactory.class);
+        doReturn(BasicTypes.BASIC_METACARD.getAttributeDescriptors()).when(mockFactory)
+                .getSupportedAttributeDescriptors();
+        List mockFactories = ImmutableList.of(mockFactory);
+
+        metacardTypeRegister = new MetacardTypeRegister() {
+            @Override
+            public BundleContext getContext() {
+                return mock(BundleContext.class);
+            }
+        };
+    }
+
+    @Test
+    public void testBind() throws Exception {
```

Comment:
So this test is really just checking that the registered metacard type has the same `AttributeDescriptors` as `BASIC_METACARD`? If that's the case you can just `assertThat(results, is(BasicTypes.BASIC_METACARD.getAttributeDescriptors()))`.